charts/redis-ha/README.md

```diff
@@ -211,6 +211,9 @@ The following table lists the configurable parameters of the Redis chart and the
 | `restore.s3.region`       | Restore init container - AWS AWS_REGION to access restore.s3.source                                                                                                                                      |``|
 | `restore.ssh.source`      | Restore init container - SSH scp location of dump - i.e. user@server:/path/dump.rdb                                                                                                                      | `false`                                                                                    |
 | `restore.ssh.key`         | Restore init container - SSH private key to scp restore.ssh.source to init container. Key should be in one line separated with \n. i.e. -----BEGIN RSA PRIVATE KEY-----\n...\n...\n-----END RSA PRIVATE KEY----- |``                                                                                 |
+| `extraContainers`         | Extra containers to include in StatefulSet                                                                                                                                                               |`[]`|
+| `extraInitContainers`     | Extra init containers to include in StatefulSet                                                                                                                                                          |`[]`|
+| `extraVolumes`            | Extra volumes to include in StatefulSet                                                                                                                                                                  |`[]`|
```

Comment:
@silvpol Don't we also need `extraVolumeMounts`? 

Because right now, your assuming people only want to mount the `extraVolumes` on their `extraContainer` / `extraInitContainer`

We should probably also give the option to include extra volumeMounts in the Redis and Sentinel containers (which is another reason why separation of these things would be nice).

Reply:
@DandyDeveloper I have added the extra volume mounts and bumped the chart version

I'm not convinced about putting containers and volumes under redis and sentinel. Despite having them under individal sections, they will still be in pod scope so you effectively end up with duplicated code that adds them in the same place one after other. Let me know if you want to go ahead anyway.

Reply:
@silvpol I agree completely. Its something to consider if we move forward with your idea of separating them out. I'm still in the process of reading your suggestion / links.